webapp/src/lang.tsx

```diff
@@ -120,7 +120,7 @@ export class LanguagePicker extends data.Component<ISettingsProps, LanguagesStat
                 closeOnEscape
             >
                 <div id="langmodal">
-                    <div id="availablelocales" className="ui cards centered" role="listbox">
+                    <div id="availablelocales" className="ui cards centered" role="list" aria-label={lf("List of available languages")}>
```

Comment:
fixes https://github.com/microsoft/pxt-microbit/issues/3663 (changes role to "list", children to "listitem"), fixes https://github.com/microsoft/pxt-microbit/issues/3662 (adds label)